products/gitlab.md

```diff
@@ -16,77 +16,92 @@ auto:
   regex: '^v?(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)-ee?$'
   git: https://gitlab.com/gitlab-org/gitlab.git
 releases:
-  - releaseCycle: "14.10"
-    release: 2022-04-22
+-   releaseCycle: "14.10"
+    release: 2022-04-21
```

Comment:
Interesting find: GitLab tags the releases on 21st of every month, but announcements are made the next day